packages/node/src/utils/flush-chunks.ts

```diff
@@ -196,7 +195,7 @@ const processChunk = async (chunk, shareMap, hostStats) => {
 export const flushChunks = async () => {
   const hostStats = loadHostStats();
   const shareMap = createShareMap();
-
+  console.log('collected chunks', usedChunks);
```

Comment:
If this is just a temporary test, it should be removed before merging into the main branch

Reply:
Yes will remove